jetty-server/src/main/java/org/eclipse/jetty/server/handler/InetAccessHandler.java

```diff
@@ -124,12 +149,17 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
      * Checks if specified address and request are allowed by current InetAddress rules.
      *
      * @param address the inetAddress to check
-     * @param request the request to check
+     * @param baseRequest the base request to check
+     * @param request the HttpServletRequest request to check
      * @return true if inetAddress and request are allowed
      */
-    protected boolean isAllowed(InetAddress address, HttpServletRequest request)
+    protected boolean isAllowed(InetAddress address, Request baseRequest, HttpServletRequest request)
     {
-        boolean allowed = _set.test(address);
+        String connectorName = baseRequest.getHttpChannel().getConnector().getName();
+
+        boolean allowed = (connectorName != null &&
+          (!_connectorNames.getIncluded().isEmpty() || !_connectorNames.getExcluded().isEmpty()) &&
```

Comment:
Do you really need these `isEmpty` checks?   I think if `_connectorNames` is empty then all strings will pass the test.  The code for test is:
```java
    public boolean test(P t)
    {
        if (!_includes.isEmpty() && !_includePredicate.test(t))
            return false;
        return !_excludePredicate.test(t);
    }
```
So if empty, this boils down to return `!excludes.contains(value)`, which is true.

Reply:
done.